boot/zephyr/prj.conf

```diff
@@ -31,7 +31,7 @@ CONFIG_FPROTECT=y
 # CONFIG_I2C is not set
 
 CONFIG_LOG=y
-CONFIG_LOG_MINIMAL=y
```

Comment:
Shouldn't comment state `former CONFIG_LOG_MINIMAL` ?

Reply:
^^ yes I should, but this comes from the upstream, my fault.